LEAF_Request_Portal/admin/templates/mod_form.tpl

```diff
@@ -1424,20 +1434,19 @@ function getForm(indicatorID, series) {
     });
 
     dialog.setSaveHandler(function() {
-    	let isRequired = $('#required').is(':checked') ? 1 : 0;
-        let isSensitive = $('#sensitive').is(':checked') ? 1 : 0;
-    	let isDisabled = $('#disabled').is(':checked') ? 1 : 0;
-        if (isSensitive === 1) {
+        /*the below values are used by the indicators table*/
+    	let requiredIndicator = $('#required').is(':checked') ? 1 : 0;
+        let sensitiveIndicator = $('#sensitive').is(':checked') ? 1 : 0;
```

Comment:
Found one of your lint red-marks.  ;-)  It will have conniptions if indenting isn't "correct".